### PR TITLE
feat: Translations!

### PR DESCRIPTION
**Pull request by AverageHelper**
I want to check these separately from the rest of #113.

### CODE REVIEW

File: src/locales/hu.json [hunk 1]
```diff
@@ -7,7 +7,16 @@
...
+			"name": "cooldown",
+			"responses": {
+				"blacklisted": "Nem küldhet be egyetlen dalt sem. Ön feketelistára került. Sajnálom",
+				"immediately": "Azonnal beküldheted!",
+				"later": "{relative} belül, helyi idő szerint {absolute} kor lehet benyújtani",
+				"submissions-all-exhausted": "Felhasználta az összes kijelölt beküldési helyet!",
+				"submissions-1-exhausted": "Felhasználta az egyetlen beadványát az éjszakára!",
+				"submissions-2-exhausted": "Mindkét beadványát felhasználta az éjszakára!",
+				"submissions-all-ct-exhausted": "Mind a {count} beadványát felhasználta az éjszakára!"
+			}
```
Comment (karcsesz) [inline src/locales/hu.json:19]:
```suggestion
			"responses": {
				"blacklisted": "Tiltólistára került, úgyhogy nem küldhet be egyetlen dalt sem. Sajnálom.",
				"immediately": "Akár most is küldhet!",
				"later": "{relative}, helyi idő szerint {absolute}-kor tud beküldeni.",
				"submissions-all-exhausted": "Felhasználta az összes beküldési lehetőségét.",
				"submissions-1-exhausted": "Felhasználta az egyetlen beküldési lehetőségét erre az alkalomra.",
				"submissions-2-exhausted": "Mindkét beküldési lehetőségét felhasználta erre az alkalomra.",
				"submissions-all-ct-exhausted": "Mind a {count} beküldési lehetőségét felhasználta erre az alkalomra."
			}
```

File: src/locales/hu.json [hunk 2]
```diff
@@ -24,15 +33,75 @@
...
+				"entry-duration-max": {
+					"description": "A beküldött dal maximális időtartama.",
+					"name": "Max. dal hossza"
+				},
+				"entry-duration-min": {
+					"description": "A beküldött dal minimális időtartama.",
+					"name": "Min. dal hossza"
+				},
+				"queue-duration": {
+					"description": "Az a maximális időtartam, ameddig a várólista eltarthat, ha az összes bejegyzést végponttól végéig lejátszaná. A várólista automatikusan bezárul, ha a beküldés miatt a sor túllépi ezt a korlátot.",
```
Comment (karcsesz) [inline src/locales/hu.json:55]:
```suggestion
					"description": "Az a maximális időtartam, amennyit a várólistán lévő dalok kitölthetnek, ha az összeset egymás után kezdettől végéig lejátszaná. A várólista automatikusan lezárul, ha egy beküldött dal miatt a lista hossza túllépi ezt a korlátot.",
```

File: src/locales/hu.json [hunk 3]
```diff
@@ -24,15 +33,75 @@
...
+					"name": "Teljes várólista hossza"
+				}
+			},
+			"responses": {
+				"infinite": "végtelen",
+				"none": "semmi",
+				"title": "Sorkorlátok",
+				"use-cooldown-cmd": "A {cooldown} segítségével megtekintheti a lehűlési időt"
```
Comment (karcsesz) [inline src/locales/hu.json:63]:
```suggestion
				"title": "Beküldési korlátok",
				"use-cooldown-cmd": "A {cooldown} segítségével megtekintheti, mennyi idő múlva tud újra beküldeni"
```

File: src/locales/hu.json [hunk 4]
```diff
@@ -24,15 +33,75 @@
...
-			"name": "mitjátszik"
+			"name": "mitjátszik",
+			"responses": {
+				"almost-sure": "85%-ban biztos vagyok benne, hogy ez {usermention} beadványa: {url}",
+				"dj-says": "A DJ azt mondta nekem, hogy ez {usermention} beadványa: {url}",
+				"good-chance": "Jó esély van rá, hogy ez {usermention} beadványa: {url}",
+				"i-think": "Szerintem ez {usermention} beadványa: {url}",
+				"likely": "Nagyon valószínű, hogy ez {usermention} beadványa: {url}",
+				"no-queue": "Jelenleg nincs sor beállítva, így semmi sem játszik le.",
+				"no-song": "Valószínűleg most nem játszik semmi. (Ha van, nem hallom)"
```
Comment (karcsesz) [inline src/locales/hu.json:76]:
```suggestion
				"almost-sure": "Úgy 85%-ban vagyok benne biztos, hogy ezt {usermention} küldte be: {url}",
				"dj-says": "Megkérdeztem a DJ-t, szerinte ezt {usermention} küldte be: {url}",
				"good-chance": "Jó esély van rá, hogy ez {usermention}-é: {url}",
				"i-think": "Szerintem ezt {usermention} küldte be: {url}",
				"likely": "Nagyon valószínű, hogy ezt {usermention} küldte be: {url}",
				"no-queue": "Jelenleg nincs sor itt beállítva, így semmi sincs a lemezjátszón.",
				"no-song": "Most semmi nem hallható. (Ha mégis, akkor beszélnem kell az orvosommal)"
```

File: src/locales/hu.json [hunk 5]
```diff
@@ -24,15 +33,75 @@
...
+			"description": "Adminisztratív parancsok a dalsor kezeléséhez.",
+			"name": "quo",
+			"options": {
+				"open": {
+					"description": "Kezdje el elfogadni a dalkérelmeket a sorba.",
+					"name": "nyisd",
+					"responses": {
+						"ack": "Elismert",
+						"already-open": "Már nyílt a sor!",
+						"no-queue": "Nincs sor a nyitáshoz. Beállítottál már egyet?",
```
Comment (karcsesz) [inline src/locales/hu.json:96]:
```suggestion
					"description": "Elkezdek dalkéréseket fogadni.",
					"name": "nyisd",
					"responses": {
						"ack": "Rendben!",
						"already-open": "Már nyitva van a sor!",
						"no-queue": "Nincs megnyitható sor. Beállítottál már egyet?",
```

File: src/locales/hu.json [hunk 6]
```diff
@@ -64,6 +133,7 @@
 				"rejections": {
 					"allotment-expended": "Felhasználta mind a(z) {max} lehetőségét.",
 					"cooldown": "Már küldött be dalt az elmúlt {duration}-ban/ben. Várnia kell {remaining}-(e)t, mielőtt újra küldhet.",
+					"queue-channel-submission-not-implemented": "A dalok lekérése a várólista csatornában még nem valósult meg.",
```
Comment (karcsesz) [inline src/locales/hu.json:136]:
```suggestion
					"queue-channel-submission-not-implemented": "Még nem vagyok képes a várólistából dalkéréseket fogadni.",
```

File: src/locales/hu.json (hunk 4 above)
Comment (AverageHelper) [reply src/locales/hu.json:76]: Screenshot 2023-03-09 at 09 13 36

> I need to talk to my doctor

I love this! 😆

File: src/locales/hu.json [hunk 7]
```diff
@@ -80,7 +150,16 @@
...
-			"name": "statisztika"
+			"name": "statisztika",
+			"responses": {
+				"title": "Személyes statisztika",
+				"average-sub-duration": "Beküldésének átlagos időtartama",
+				"blacklisted": "Feketelistás",
+				"cooldown-remaining": "A lehűlésből hátralévő idő",
+				"queue-empty": "A sor üres. Jelenleg nincsenek statisztikái.",
+				"total-subs": "Öntől származó beküldések összesen",
+				"value-limit": "{value} (korlát {limit})"
```
Comment (karcsesz) [inline src/locales/hu.json:161]:
```suggestion
				"average-sub-duration": "Beküldött dalainak átlagos időtartama",
				"blacklisted": "Tiltólistán van",
				"cooldown-remaining": "A következő beküldési lehetőségéig hátralévő idő",
				"queue-empty": "A sor üres. Jelenleg nincsenek statisztikái.",
				"total-subs": "Összesen beküldött dalok",
				"value-limit": "{value} (korlát {limit})"
```

File: src/locales/fr.json [hunk 8]
```diff
@@ -7,7 +7,16 @@
...
-			"name": "cooldown"
+			"name": "cooldown",
+			"responses": {
+				"blacklisted": "Vous pouvez soumettre une fois que vous êtes retiré de la liste noire. Je suis désolé",
+				"immediately": "Vous pouvez soumettre dès maintenant !",
+				"later": "Vous pouvez soumettre en {relative}, à {absolute} heure locale",
+				"submissions-all-exhausted": "Vous avez utilisé toutes vos soumissions pour la nuit!",
+				"submissions-1-exhausted": "Vous avez utilisé votre seule soumission pour la nuit!",
+				"submissions-2-exhausted": "Vous avez utilisé vos deux soumissions pour la nuit!",
+				"submissions-all-ct-exhausted": "Vous avez utilisé vos {count} soumissions pour la soirée!"
```
Comment (vayandas) [inline src/locales/fr.json:18]: replace soirée by nuit for consistency

File: src/locales/fr.json [hunk 9]
```diff
@@ -24,15 +33,75 @@
...
+			"description": "Commandes administratives pour gérer la file d'attente des chansons.",
+			"name": "quo",
+			"options": {
+				"open": {
+					"description": "Commencez à accepter les demandes de chansons dans la file d'attente.",
+					"name": "ouvrir",
+					"responses": {
+						"ack": "J'accuse réception",
+						"already-open": "La file d'attente est déjà ouverte!",
+						"no-queue": "Il n'y a pas de file d'attente à ouvrir. En avez-vous déjà créé un?",
```
Comment (vayandas) [inline src/locales/fr.json:96]: créée une instead of créé un (french gendered language hit again!)

File: src/locales/fr.json [hunk 10]
```diff
@@ -7,7 +7,16 @@
 	"commands": {
 		"cooldown": {
 			"description": "Découvrez quand vous pourrez soumettre à nouveau.",
-			"name": "cooldown"
+			"name": "cooldown",
+			"responses": {
+				"blacklisted": "Vous pouvez soumettre une fois que vous êtes retiré de la liste noire. Je suis désolé",
```
Comment (vayandas) [inline src/locales/fr.json:12]: désolée instead of désolé (yay gendered language)

**Review by vayandas — CHANGES_REQUESTED** [review]

**Pull request merged**